src/node_file.cc

```diff
@@ -1398,7 +1400,6 @@ int MKDirpAsync(uv_loop_t* loop,
             }
             // verify that the path pointed to is actually a directory.
             if (err == 0 && !S_ISDIR(req->statbuf.st_mode)) err = UV_EEXIST;
-            uv_fs_req_cleanup(req);
```

Comment:
We were calling `uv_fs_req_cleanup` prematurely on Windows systems, and freeing `req->path`.

`uv_fs_req_cleanup` is called in `FSReqAfterScope::~FSReqAfterScope()`.